File: wellness/wellness.Model/Record/Record.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace wellness.Model.Record
{
    public class Record
    {
        public int Id { get; set; }

        public String? EntryDate { get; set; }

        public String? LeaveEntryDate { get; set; }

        public int UserId { get; set; }

        public string FirstName { get; set; } = null!;

        public string LastName { get; set; } = null!;
        public string? Phone { get; set; }
        public string UserName { get; set; } = null!;

    }
}

File: wellness/UI/desktop/lib/models/record.dart
import 'package:json_annotation/json_annotation.dart';

part 'record.g.dart';

@JsonSerializable()
class Records {
  final int id;
  final String? entryDate;
  final String? leaveEntryDate;
  final int userId;
  final String firstName;
  final String lastName;
  final String phone;
  final String userName;

  factory Records.fromJson(Map<String, dynamic> json) =>
      _$RecordsFromJson(json);

  Records(this.id, this.entryDate, this.leaveEntryDate, this.userId,
      this.firstName, this.lastName, this.phone, this.userName);

  Map<String, dynamic> toJson() => _$RecordsToJson(this);
}
